src/impi/core.clj

```diff
@@ -0,0 +1,12 @@
+(ns impi.core)
+
+(defmacro add-props! [& {:as coll}]
```

Comment:
Well, right now, Pixi 4.0 doesn't work with Closure's advanced compilation for some reason, so renaming keys doesn't work regardless :)

Your point is a good one, but I think I'd still rather increase the filesize by a small amount than commit to the wrong abstraction. I don't think macros are necessarily a bad idea, but I think the project is too young, and we can do without them for a while.